composer.json

```diff
@@ -206,7 +206,7 @@
                 "Custom blocks are no longer displayed and no longer appear in the custom block library - https://www.drupal.org/project/drupal/issues/2997898#comment-12832813": "https://www.drupal.org/files/issues/2018-10-27/2997898-21.patch",
                 "Make the DateTime input format and descriptive text consistent - https://www.drupal.org/project/drupal/issues/2791693#comment-13024583": "https://www.drupal.org/files/issues/2019-03-16/2791693-38.patch",
                 "Poor performance when using moderation state views filter - https://www.drupal.org/project/drupal/issues/3097303#comment-13559283": "https://www.drupal.org/files/issues/2020-04-17/3097303-9.patch",
-                "Refactor Xss::attributes() to allow filtering of style attribute values - https://www.drupal.org/project/drupal/issues/3109650#comment-14151845": "https://www.drupal.org/files/issues/2021-07-02/refactor_xss_attributes-3109650-38.patch"
+                "Refactor Xss::attributes() to allow filtering of style attribute values - https://www.drupal.org/project/drupal/issues/3109650#comment-14151845": "https://www.drupal.org/files/issues/2021-04-22/refactor_xss_attributes-3109650-27.patch"
```

Comment:
There is a warning with that patch

    If you have any Wysiwyg editors that are open to non-trusted sources, you should NOT add this patch, as it would introduce a XSS security issue!

Does this present a risk to us or anyone else using the tide distribution?

Reply:
our editors are basically trusted 😬  . 
It does present potential risks. it was introduced by https://digital-engagement.atlassian.net/browse/SDPA-4334